aries-site/src/pages/components/pagination.mdx

```diff
@@ -1,42 +1,69 @@
 import { Example } from '../../layouts';
 import { ExampleImagePreview } from '../../components';
-import { PaginationCardsExample, PaginationExample } from '../../examples';
+import {
+  PaginationAnatomy,
+  PaginationCardsExample,
+  PaginationExample,
+  ToggleGroupViews,
+} from '../../examples';
 
 <Example
   componentName="Pagination"
   code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/pagination/PaginationTableExample.js"
   docs="https://v2.grommet.io/pagination?theme=hpe#props"
-  figma="https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?type=design&node-id=26125-326&mode=design"
+  figma="https://www.figma.com/design/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=26125-326"
   grommetSource="https://github.com/grommet/grommet/blob/master/src/js/components/Pagination/Pagination.js"
+  showResponsiveControls={['laptop', 'mobile']}
+  height={{ max: 'small' }}
```

Comment:
Seems strange to have a "max" height here, but we could set a `min`?
```suggestion
  height={{ min: 'xsmall' }}
```

Reply:
I think I meant `min`